server/mediapublic/__init__.py

```diff
@@ -40,4 +43,13 @@ def main(global_config, **settings):
 
     config.include('velruse.providers.twitter')
 
+    config.add_view(mp_exc.handle_exceptions, context=Exception,
+                    permission=security.NO_PERMISSION_REQUIRED)
+    config.add_view(hooks.handle_exceptions,
```

Comment:
If we (you) don't have a need for it right now, I don't see a need to add it to this PR.  Are you okay with that @GabeIsman ?